pyroma/projectdata.py

```diff
@@ -4,6 +4,7 @@
 import os
 import pathlib
 import re
+from importlib.metadata import metadata
```

Comment:
Would it be worth importing this under an alias, rather than having both `metadata` the import and `metadata` the variable in the same file?

Reply:
I'm fine with calling the variable just "md", that's what we use for metadata at my work anyway. :-)